frontend/components/profile/ViewProfile.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { View, Text, Button, Image, StyleSheet, Dimensions } from 'react-native';
```

Comment:
Need to import some elements from react native elements, add:
`import { Card, Header, Chip } from '@rneui/themed';`